compute/cker/include/cker/operation/optimized/DepthwiseConvUint8.h

```diff
@@ -1253,7 +1255,7 @@ template <> struct QuantizedDepthwiseConvKernel<true, 1, 16>
     {
       uint8_t input_u8 = *input_ptr;
       input_ptr += input_ptr_increment;
-      uint16_t input = static_cast<int16_t>(input_u8 + input_offset);
+      int16_t input = static_cast<int16_t>(input_u8 + input_offset);
```

Comment:
```suggestion
      int16_t input = static_cast<int16_t>(input_u8) + input_offset;
```

Reply:
not sure as have little understand of the codes... but if change is OK, then please update eothers too